students/briggsm/session04/mb_trigram.py

```diff
@@ -0,0 +1,54 @@
+import random as rand
+
+def create_corpus(infile):
+    '''Input: a filename
+       Output: a list of words.'''
+    corpus = []
+    with open(infile) as lines:
+        lines.readline()
+        for line in lines:
+            line = line.strip()
+            words = line.split(" ")
+            for i in range(len(words)):
+                corpus.append(words[i])
+    return corpus
+
+def create_trigram(textlist):
+    '''Input: a list of words from a text body.
+       Output: a dictionary of two words in sequence and the word after.'''
+    trigram = {}
+    for w in range(len(textlist[:-2])):
+        firstword = textlist[w]
+        secondword = textlist[w+1]
+        thirdword = textlist[w+2]
+        trigram[thirdword] = (firstword, secondword)
+    return trigram
+
+def get_values(trigram):
+    '''Input: a dictionary.
+       Output: a list of keys.'''
+    words = []
+    for w in trigram.keys():
+        words.append(w)
+    return words
+
+def get_random_item(listofwords):
+    '''Input: a list of items.
+       Output: a random item as a string.'''
+    choose = rand.randint(0,len(listofwords))
+    return str(listofwords[choose])
```

Comment:
this works, but there's the:

`random.choice(listofwords)`

function which does the same thing.